EventHubs/src/Deploy.parameters.ps1

```diff
@@ -1,7 +1,24 @@
-param ([Parameter(Mandatory = $true)][string]$Lifecycle, [string] $LifecyclePrefix)
+param([string] $Lifecycle, [string] $LifecycleSuffix)
 
-[System.Reflection.Assembly]::LoadFrom("$PSScriptRoot\Gcm.ConfigurationLoader.dll") | Out-Null
+Connect-AzureAccount | Out-Null
 
-$obj = [Gcm.ConfigurationLoader.ConfigurationLoader]::LoadHashtable("$PSScriptRoot/Config/deploy.app.json", "$PSScriptRoot/Config/deploy.variables.json", [string[]]@("default", $Lifecycle))
+# Fetch the App Insights Instrumentation Key programmatically for each environment
```

Comment:
Add add to the comment: 
At the time of writing, for the gcmdev/gcmtst tenants, the app insights instances are still using gcm- and not gcmdev-/gcmtst- as tenant prefixes.